pkg/cli/root.go

```diff
@@ -306,6 +306,24 @@ func validateCacheConfig(cfg *config.Config) error {
 	return nil
 }
 
+func validateExtensionsConfig(cfg *config.Config) {
+	//nolint:lll
+	if cfg.Storage.StorageDriver != nil && cfg.Extensions != nil && cfg.Extensions.Search != nil && cfg.Extensions.Search.CVE != nil {
```

Comment:
do not modify the internal config :D, will create issues with the configloader.

just return an error, look at sync+s3 combination which also panics

Reply:
Added